### PR TITLE
feat : created document page layout, alert and avatar components

### PR DESCRIPTION
**Pull request by samyukthasudhakar**
Developed the following features as part of this pull request :

- documentation page layout with side menu bar
- alert components
- avatar components

Preview Link : https://alert-and-avatar.netlify.app/docs.html

Looking forward to get the changes reviewed and receive suggestions to improve.

### CODE REVIEW

**Comment by Dev-AkshatGupta** [comment]
Its seems and works amazing but You have not mentioned the code for the Avatars in documentation.
Keep on doing the good work :)

File: styles/colors.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+:root{
+
+    /*background colors*/
+    --clr-blue:#B9eAed;
```
Comment (sainath7878) [inline styles/colors.css:4]: Naming of the color variables could be better.

File: styles/components/alerts.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+.alert-danger{
+    background-color: var(--clr-red);
+}
+
+.alert-warning{
+    background-color: var(--clr-orange);
+}
+
+.alert-cancel-btn{
+    float:right;
```
Comment (sainath7878) [inline styles/components/alerts.css]: Float is old school. You can use other alternatives. Personal opinion though :)

File: styles/components/component.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+.component-variant-container{
+    margin:1rem 0;
+}
+
+.component-variant-heading{
+    margin:1rem 0;
+}
+.component-variant-content{
+    padding:1rem;
+    display:flex;
```
Comment (sainath7878) [inline styles/components/component.css:27]: As flex is used many a times in your code. You can make this a utility class and use wherever required.

**Comment by samyukthasudhakar** [comment]
> Its seems and works amazing but You have not mentioned the code for the Avatars in documentation. Keep on doing the good work :)

Thank you so much for noticing this, totally slipped my mind. I have added the code blocks for avatars now.

File: styles/colors.css (hunk 1 above)
Comment (samyukthasudhakar) [reply styles/colors.css:4]: Sure will keep in mind and make changes.

File: styles/components/component.css (hunk 3 above)
Comment (samyukthasudhakar) [reply styles/components/component.css:27]: Created a utils for flex layout types and used styling from there after your recommendation. Thanks 😊

File: styles/components/alerts.css (hunk 2 above)
Comment (samyukthasudhakar) [reply styles/components/alerts.css]: Changed the implementation to use flex instead of float 👍🏽

File: styles/components/avatar.css [hunk 4]
```diff
@@ -0,0 +1,54 @@
+.avatar-holder{
```
Comment (vrx29) [inline styles/components/avatar.css:1]: Try using avatar-container instead of holder. As it is more understandable.

**Review by RakshaPawar108 — COMMENTED** [review]
You can try making the page responsive by adding a few media queries  to your CSS. The rest of the code looks good and clean too. Love the UI design on desktop. Great work!!

File: styles/components/avatar.css (hunk 4 above)
Comment (samyukthasudhakar) [reply styles/components/avatar.css:1]: Sure will do that !

**Comment by samyukthasudhakar** [comment]
> You can try making the page responsive by adding a few media queries to your CSS. The rest of the code looks good and clean too. Love the UI design on desktop. Great work!!

I've not yet worked on making the site responsive, but will definitely keep in mind. Thank you so much 😄

**Pull request merged**